data/human/paradise world jobs.txt

```diff
@@ -478,13 +478,10 @@ mission "Paradise Job: Retirees"
 	passengers 7 12 .6
 	cargo "treasured possessions" 7 12 .2
 	to offer
-		random < 10
-		"passenger space" > 30
-		"cargo space" > 120
+		"passenger space" > 15
+		"cargo space" > 40
 	on offer
 		require "Luxury Accommodations"
-	source
-		attributes "frontier" "dirt belt" "south" "farming" "mining" "rim" "forest" "urban"
```

Comment:
Was this intentional?

Reply:
Whoops, thanks for catching that, removed them temporarily to test it, since that mission has very low cargo/passengers I had to triple-check myself.